vizro-core/docs/pages/user-guides/assets.md

```diff
@@ -155,24 +184,35 @@ Suppose you want to hide the page title on one page only. Here's how you can ach
 
 
 ## Overwrite CSS properties in selective components
-To adjust CSS properties for specific components, like altering the appearance of a selected [Card][vizro.models.Card] rather than all Cards,
-you'll need to target the correct CSS selector based on the component hierarchy.
+To adjust CSS properties for specific components, like altering the appearance of a selected [`Card`][vizro.models.Card] rather than all `Card`s,
+you need to use a CSS selector that targets the right CSS property.
 If you're unfamiliar with CSS selectors, you can refer to this [tutorial](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors/Selector_structure) for guidance.
 
+Let's say we want to change the background and font-color of a specific `Card`.
+
 Here's how you can do it:
 
 1. Assign a unique `id` to the relevant `Card`, for example: `Card(id="custom-card", ...)`
-2. Review the component's source code to find the appropriate CSS class or element you need to target.
+2. Review the HTML document structure in your browser to find the appropriate CSS class or element you need to target.
+We recommend using the [Chrome DevTools](https://developer.chrome.com/docs/devtools/) to do that.
+For guidance on how to get the CSS selector in Chrome, see this [thread](https://stackoverflow.com/questions/4500572/how-can-i-get-the-css-selector-in-chrome).
 
 It's essential to understand the relationship between the targeted CSS class or element and the component assigned the `id`, for example:
 
-* The `id` is provided to the `dcc.Markdown` component inside the `Card`.
-* The `dcc.Markdown` component is wrapped inside a parent container with the class name `"card"`.
-* The card text is wrapped inside a `<p>` element that is a child of the `dcc.Markdown` component.
+<!-- vale off -->
+```python title="HTML structure of a `Card`"
+<div class="card card">
```

Comment:
Is `card card` meant to be repeated? 🤔

Reply:
No 😄 One comes automatically from the dbc component, the other one we added. I've removed it now: https://github.com/mckinsey/vizro/pull/462/commits/a0e903e14ce01a1eef3a21403398659671ba4bb3